src/core/network.js

```diff
@@ -75,6 +75,24 @@ var NetworkManager = (function NetworkManagerClosure() {
     return array.buffer;
   }
 
+//#if (FIREFOX || MOZCENTRAL)
+//var supportsMozChunked = true;
+//#endif
+//#if GENERIC
```

Comment:
That's a good idea. Let's use that logic for FIREFOX as well, e.g. use supportMozChunked only when `!(FIREFOX || MOZCENTRAL || CHROMIUM)`

`#elseif` shall be expressed in terms of `#if CASE1`, `#if CASE2`, `#if !(CASE1 || CASE2)`